### PR TITLE
Blog rewrite rules

### LINKED ISSUE #635: Shut down public view of WC blog
Phase 1: Continue to use blog for content management, use it as a redirection service
- [x] In Drupal, replace all blog references with corresponding `next.` link
~~- [ ] Redirect DNS~~

### PR DESCRIPTION
**Pull request by jamesgorrie**
## What is this PR trying to achieve?
Getting more ready for the big switcheroo.

This has made the category and tag redirects correct, and also allows feed (RSS) links to proxy through to Wordpress.

Once this is done, we take over the URL - and voi la.

Fixes #635

### CODE REVIEW

File: nginx-proxy/nginx.conf [hunk 1]
```diff
@@ -8,21 +8,24 @@ http {
...
 
     # tags
-    rewrite ^/tag\/(.*)/$ https://next.wellcomecollection.org/articles?q=tag:$1 redirect;
+    rewrite ^/tag\/(.*)/?$ https://next.wellcomecollection.org/articles?q=tags:$1 redirect;
 
     # pages
-    rewrite ^/about/$ https://wellcomecollection.org/what-we-do/about-wellcome-collection redirect;
-    rewrite ^/about/terms-and-conditions/$ https://wellcome.ac.uk/about-us/terms-use redirect;
+    rewrite ^/about/?$ https://wellcomecollection.org/what-we-do/about-wellcome-collection redirect;
+    rewrite ^/about/terms-and-conditions/?$ https://wellcome.ac.uk/about-us/terms-use redirect;
```
Comment (alfraser86) [inline nginx-proxy/nginx.conf]: Is this really needed?

Comment (jamesgorrie) [inline nginx-proxy/nginx.conf]: Yes? Both links exist and should point to different places.
See the [conversation](https://github.com/wellcometrust/wellcomecollection.org/issues/635) and [conclusion](https://github.com/wellcometrust/wellcomecollection.org/issues/635#issuecomment-289478298).

Comment (jamesgorrie) [inline nginx-proxy/nginx.conf]: I would have imaged that we would have our own pages (collection content is definitely going to have some singing and dancing around open access) - but I've seen us point to Wellcome Trust from other locations as well.

Comment (alfraser86) [inline nginx-proxy/nginx.conf]: Yeah the pointing to wellcome trust is fine. But i'm not sure why if you have a link going to a different site you would need a redirect rule.

Comment (jamesgorrie) [inline nginx-proxy/nginx.conf]: We have the URL:
`https://blog.wellcomecollection.org/about/terms-and-conditions/`

Which will redirect to:
`https://wellcome.ac.uk/about-us/terms-use redirect`

**Comment by jamesgorrie** [comment]
@gestchild @davidpmccormick Fancy taking a peek at this?

Basically it's saying do the redirection stuff, except for anything pointing to `*/feed/` -> just point that to Wordpress.

**Pull request merged**